app/javascript/chat/chat.jsx

```diff
@@ -558,6 +559,19 @@ export default class Chat extends Component {
         e.preventDefault();
       }
     }
+    if (upArrowPressed && messageIsEmpty) {
```

Comment:
>  I used the up arrow key for the shortcut.

Not sure if that's required to delete

Reply:
Are you referring to the check to see if the message is empty?

Reply:
I'm referring to the feature of showing delete popup with up arrow key.

Reply:
it was requested in the issue to have a shortcut to delete the last message

Reply:
Oh then it's fine. Great Job.

Reply:
Thanks!